frontend/ui/widget/keyvaluepage.lua

```diff
@@ -113,7 +113,8 @@ end
 local KeyValueItem = InputContainer:new{
     key = nil,
     value = nil,
-    cface = Font:getFace("cfont"),
+    cface = Font:getFace("cfont", 22),
```

Comment:
Font size should be wrapped by `Screen:scaleBySize`. And I suggest changing the default size in `Font` module so we get a consistent experience across the app.